README.md

```diff
@@ -105,11 +106,21 @@ sudo dnf install ani-cli
 
 #### Arch
 
+Build and install from the AUR: 
+```sh
+yay -S ani-cli
+```
 Also consider ani-cli-git
 
+#### OpenSuse Tumbleweed
```

Comment:
I would assume this works in opensuse leap as well, whats up with that?

Reply:
Absolutely no idea. I haven't even tested the package on tumbleweed, I just assume it works... Worst case I make a chroot for the different versions of leap too.